source/fundamentals/logging.txt

```diff
@@ -0,0 +1,273 @@
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+In this guide you can learn how to do the following:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+To set up a logger for your project, you need to add some dependencies.
+
+The MongoDB Java driver uses the Simple Logging Facade For Java (SLF4J).
+SLF4J allows you to specify your logging framework of choice at deployment time.
+For more information on SLF4J,
+`see the SLF4J documentation <http://www.slf4j.org/>`__.
+
+To set up a logger you must add the ``slf4j-api`` package, a logging framework, and a
+**binding** to your project's dependency list. A binding is a piece of code that
+connects the ``slf4j-api`` package with a logging framework. The two most
+popular logging frameworks used with ``slf4j-api`` are Log4j2 and Logback.
+
+Setting up a logger is optional. When you start your application the MongoDB
+Java driver looks for the ``slf4j-api`` package in your classpath. If the driver
+can't find ``slf4j-api``, the driver logs a single warning with
+``java.util.logging`` and disables all further logging.
+
+Here is the error the driver outputs if it cannot find the ``slf4j-api`` package:
+
+.. code-block:: none
+
+   WARNING: SLF4J not found on the classpath.  Logging is disabled for the 'org.mongodb.driver' component
+
+
+Below is an example showing how to add SLF4J dependencies to your
+project and set up your logger using either Logback or Log4j2. The examples uses Maven.
+
+.. tabs::
+
+   .. tab:: Logback
+      :tabid: logback-binding
+
+      If you add these dependencies to your ``pom.xml`` file, the MongoDB Java
+      driver uses the Logback framework to output your logs.
+
+      .. code-block:: xml
+
+         <!-- SLF4J API -->
+         <dependency>
+           <groupId>org.slf4j</groupId>
+           <artifactId>slf4j-api</artifactId>
+           <version>1.7.30</version>
+         </dependency>
+         <!-- This package contains an SLF4J-Logback binding -->
+         <dependency>
+           <groupId>ch.qos.logback</groupId>
+           <artifactId>logback-classic</artifactId>
+           <version>1.2.3</version>
+         </dependency>
+         <!-- Logback Core Functionality -->
+         <dependency>
+           <groupId>ch.qos.logback</groupId>
+           <artifactId>logback-core</artifactId>
+           <version>1.2.3</version>
+         </dependency>
+
+      Once you have installed the above dependencies, try connecting to a
```

Comment:
nitpick: there is no try.

In all seriousness, be prescriptive. And this seems like another step. If so, use the proper mood.

> Connect to a MongoDB instance and retrieve...

Reply:
Thanks for the note. Going forward, I will try (there is no try, but it is unlikely I will perfectly implement this feedback immediately haha) to be prescriptive in my writing. I will avoid the word "try" is step by step guides.

Fix: https://github.com/mongodb/docs-java/pull/76/commits/a9644a761d0a1f71ec9572015087f6246b69a44e